File: WeekendWarriors/Questionnaires/Questionnaires/QLS/AST/QuestionWithWidget.cs
using Questionnaires.QLS.AST.Widgets;

namespace Questionnaires.QLS.AST
{
    class QuestionWithWidget : Question
    {
        public Widget Widget{ get; }
        public QuestionWithWidget(string name, Widget widget) : base(name)
        {
            this.Widget = widget;
        }
    }
}

File: Loading/QLS/src/qls/ast/QuestionWithWidget.java
package qls.ast;

import qls.ast.widget.Widget;

public class QuestionWithWidget extends Question {

	final private Widget widget;
	
	public QuestionWithWidget(String name, Widget widget, int line) {
		super(name, line);
		this.widget = widget;
	}
	
	public void accept(StylesheetVisitor v) {
		v.visit(this);
	}

}
